vimpager

```diff
@@ -24,11 +24,13 @@ done
 project_dir="`dirname \"$link\"`"
 # END OF FIND REAL PARENT DIRECTORY
 
-. "$project_dir""/inc/prologue.sh"
-
 version="$(cd "$project_dir"; git describe) (git)"
```

Comment:
We are not POSIX jet. I think these variables should come after prologue.sh (which finds and execs the POSIX shell).